client/app/index.html

```diff
@@ -6,7 +6,6 @@
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, user-scalable=no" />
     <meta name="referrer" content="never" />
-    <base href="/" />
```

Comment:
yep for now @NSkelsey do not mind at this; i also got various issues that i now do not remember but let's keep that hashbang for now :)